debian/control

```diff
@@ -317,6 +317,7 @@ Description: MariaDB database core client binaries
 Package: mariadb-client-10.5
 Architecture: any
 Depends: debianutils (>=1.6),
+         libconfig-inifiles-perl,
```

Comment:
what uses this? (https://packages.debian.org/buster/all/libconfig-inifiles-perl/filelist)

Reply:
mytop apparently: https://salsa.debian.org/mariadb-team/mariadb-10.5/-/commit/7a748da87a23eb874d0bddb64114a795e9d07e6d